src/Spinners/LoadSpinner.tsx

```diff
@@ -0,0 +1,47 @@
+import React, {
+  FunctionComponent, ReactElement, useContext,
+} from 'react';
+import styled, { ThemeContext } from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faCircleNotch } from '@fortawesome/free-solid-svg-icons';
+import { BaseTheme } from 'Theme';
+import LoadSpinnerText, { SPINNER_TEXT } from './LoadSpinnerText';
+
+export interface LoadSpinnerProps {
+  /** Specifies the text under the spinner */
+  children: string;
+  /** Specifies the color of text under the spinner */
+  color?: SPINNER_TEXT;
```

Comment:
Can we rename this to `textColor`? and do so in the `LoadSpinnerText` component as well? I always find it better to be more specific with prop names.

Reply:
yes , this make sense